rx-android-extensions/src/main/java/com/appunite/rx/android/util/LogTransformer.java

```diff
@@ -34,6 +34,22 @@
 
         Logger DEFAULT = new Logger() {
             @Override
+            public void logSubscribe(@NonNull String tag, @Nonnull String observableTag) {
+                final java.util.logging.Logger logger = java.util.logging.Logger.getLogger(tag);
+                if (logger.isLoggable(Level.INFO)) {
+                    logger.log(Level.INFO, observableTag + " -> onSubscribe");
+                }
+            }
+
+            @Override
+            public void logUnsubscribe(@NonNull String tag, @Nonnull String observableTag) {
+                final java.util.logging.Logger logger = java.util.logging.Logger.getLogger(tag);
+                if (logger.isLoggable(Level.INFO)) {
```

Comment:
if (logger.isLoggable(Level.INFO))  is used to avoid potentially heavy toString call on release builds, logSubscribe and logUnsubscribe dont call toString on any object so this check is unnecessary